packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -125,6 +125,14 @@ protected function doTestFile(string $fixtureFilePath): void
         // write temp file
         FileSystem::write($inputFilePath, $inputFileContents);
 
+        if (! is_file($inputFilePath)) {
```

Comment:
Why shouldn't the file exists? Where is it deleted?

Reply:
Probably overlapped by tearDown 

https://github.com/rectorphp/rector-src/blob/10c36b08fa0c980bd31b3059cafa12610852ebdd/packages/Testing/PHPUnit/AbstractRectorTestCase.php#L73-L78

Reply:
Could be verified by dropping the delete. 

Do tests exist which re-use the same fixture ?

Reply:
that's temporary file with `.php` file instead of `.php.inc` so it needs to be deleted, I am thinking that internal phpunit tearDown may cause the overlap